### DISCUSSION TITLE
Help with Java 16

### DISCUSSION THREAD

**Discussion by HierarchyMan** [diamond-rip/Eden#97] [Q&A] [answered]
Fawe won't load without Java 8
Imanity spigot 3 won't load without Java 16 
But Eden hard depends on fawe and using worldedit won't let me load Eden cuz it needs fawe
Help how do I enable Eden/fawe

**Comment by Shzv** [comment]
you could remove fawe from eden via plugin.yml but i advise against it. however from the Imanityspigot 3 document java 8 is natively supported.
https://docs.imanity.dev/imanityspigot/compatibility

**Reply by HierarchyMan** [reply]
I removed it but some events won't load which is why after vigorous testing I posted this

**Reply by Shzv** [reply]
sorry but Imanity spigot doesn't suit you with java 8?

**Reply by HierarchyMan** [reply]
No it doesn't cuz I have some plugins that require Java 11 or above

**Comment by HappyAreaBean** [comment, marked as answer]
There is a fork of FAWE that with Java 17 support, e.g. https://github.com/DawningW/FastAsyncWorldedit-Legacy.
Something may be broken, since no one run through all the command to test. I still recommend staying on Java 8 if you don't want to risk it.

**Reply by HierarchyMan** [reply]
OMG ty

**Reply by HappyAreaBean** [reply]
> OMG ty

Remember to mark this as answer if it answered your question :)

**Comment by GoodestEnglish** [comment]
Fawe should load with java 8, one of my friend's server is running fawe with java 8.

**Reply by HierarchyMan** [reply]
I don't think that's what I meant. but
 i found fawe fork that works with java 16  thanks to @HappyAreaBean my issue is resolved
